sssd_test_framework/hosts/ldap.py

```diff
@@ -27,6 +27,10 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
         self._features: dict[str, bool] | None = None
+        if "ldap_service_name" in kwargs:
+            self._ldap_service_name = kwargs["ldap_service_name"]
+        else:
+            self._ldap_service_name = "dirsrv@localhost.service"
```

Comment:
`self.config.get("ldap_service_name", "dirsrv@localhost.service")`

Then you can set the value in mhc.yaml if needed.

Reply:
Done, thanks.